core/tee/fs_htree.c

```diff
@@ -646,32 +680,23 @@ TEE_Result tee_fs_htree_open(bool create, uint8_t *hash, uint32_t min_counter,
 	ht->stor_aux = stor_aux;
 
 	if (create) {
-		const struct tee_fs_htree_image dummy_head = {
-			.counter = min_counter,
-		};
-
-		res = crypto_rng_read(ht->fek, sizeof(ht->fek));
-		if (res != TEE_SUCCESS)
-			goto out;
-
-		res = tee_fs_fek_crypt(ht->uuid, TEE_MODE_ENCRYPT, ht->fek,
-				       sizeof(ht->fek), ht->head.enc_fek);
-		if (res != TEE_SUCCESS)
-			goto out;
-
-		res = init_root_node(ht);
-		if (res != TEE_SUCCESS)
-			goto out;
-
-		ht->dirty = true;
-		res = tee_fs_htree_sync_to_storage(&ht, hash, NULL);
-		if (res != TEE_SUCCESS)
-			goto out;
-		res = rpc_write_head(ht, 0, &dummy_head);
+		/* This flow need to resislience to power loss */
```

Comment:
```suggestion
		/* This flow needs to be resilient to power loss */
```
or
```suggestion
		/* This flow needs to resist to power loss */
```
I wonder if this inline comment is needed. The one added below seems enough.

Reply:
Thank you. Keeping this inline comment can help us stay aware of the risk of power loss. Also, using the term 'resilient' is preferable.